src/viking.js

```diff
@@ -1,11 +1,119 @@
-// Soldier
-function Soldier() {}
+
+function Soldier (healthArg, strengthArg) {
+	this.health = healthArg;
+	this.strength = strengthArg;
+	//this.name = nameArg; // I would pass 3 arguments to Soldier for the subclasses to inherit.
```

Comment:
The reason why we don't add the name to the soldier is because the Saxon object doesn't have a name.